test/doctool/test-doctool-html.js

```diff
@@ -98,38 +106,15 @@ testData.forEach(function(item) {
           // expected list
           assert.notEqual(actual.indexOf(expected), -1);
 
-          // GA script should not be present by default
-          assert.equal(actual.indexOf('google-analytics.com'), -1,
-                       'Google Analytics script was present');
-        }));
-    }));
-  }));
-});
-
-// testing the insertion of Google Analytics script when
-// an analytics id is provided
-testData.forEach((item) => {
```

Comment:
Building docs for multiple files concurrently made the build randomly fail. Caused by some shared internal state variables in [tools/doc/html.js#L33](https://github.com/nodejs/node/blob/master/tools/doc/html.js#L33).